_data/meltano/utilities/elementary/elementary.yml

```
@@ -0,0 +1,80 @@
+name: elementary
+label: Elementary
+domain_url: https://docs.elementary-data.com/introduction
+namespace: elementary
+pip_url: elementary-data[bigquery]=0.5.3 git+https://github.com/potloc/elementary-ext.git
+maintenance_status: active
+repo: https://github.com/elementary-data/elementary
+executable: edr
+logo_url: /assets/logos/utilities/elementary.png
+variant: elementary
+description: Elementary is an open-source data observability solution for data & analytics engineers.
+
+settings:
+- name: profiles-dir
+  description: Profiles directory path for your dbt project
+  kind: string
+  value: ${MELTANO_PROJECT_ROOT}/transform/profiles/bigquery/
```

```suggestion
  value: ${MELTANO_PROJECT_ROOT}/transform/profiles/bigquery/
```

Likely we shouldn't have a default value for the user and perhaps just have a `placeholder`. As part of `next_steps` we could have them set this.

It would be nice if `bigquery` was an available environment variable 🤔

---

@tayloramurphy I can set that up in the coming days

---

@tayloramurphy finally I opted out of specifying the adapter as a variable because it still needs to be specified in the pip url. For now the profiles-dir gives you enough flexibility, but I would be very interested in creating a workflow where when you install the package, you can directly specify the adapter which would affect the pyproject setup.